environment/src/main/resources/application.yml

```diff
@@ -89,7 +89,7 @@ environment:
     url: http://localhost:8090
     contextPath: /freeipa
     synchronizeOnStart: true
-    supportedPlatforms: AWS,AZURE,GCP
+    supportedPlatforms: AWS,AZURE,GCP,MOCK
```

Comment:
are we sure isn't this going to cause a problem on any mow env?

Reply:
on mow-envs we have defined supportedPlatforms: https://github.infra.cloudera.com/thunderhead/cloud-services-infra/search?q=supportedPlatforms